.gitignore

```diff
@@ -5,3 +5,6 @@
 .project
 .classpath
 .DS_Store
+
+*.iml
+/.idea
```

Comment:
nitpick but it may mess up other users' editors - can we add newlines at EOF where missing?